Dockerfile

```diff
@@ -1,31 +1,33 @@
-FROM python:3.6-slim
+FROM python:3.6.9-slim
 
-# add a non-privileged user for installing and running
-# the application
-RUN groupadd -g 10001 app && \
-    useradd -d /app -g 10001 -G app -M -s /bin/sh -u 10001 app
+# Set up user and group.
+ARG groupid=10001
+ARG userid=10001
 
 WORKDIR /app
+RUN groupadd --gid $groupid app && \
+    useradd -g app --uid $userid --shell /usr/sbin/nologin --create-home app
```

Comment:
This is how we did Socorro. This makes it easier for Linux developers to do development without dealing with file permission issues.